library/core/src/num/f64.rs

```diff
@@ -719,6 +719,62 @@ impl f64 {
         intrinsics::minnumf64(self, other)
     }
 
+    /// Returns the maximum of the two numbers.
```

Comment:
I think it's important that the first line here not exactly match the other max function, and that the difference be mentioned up-front ASAP, for people not so familiar with IEEE nuances.

Perhaps something like
```suggestion
    /// Returns the maximum of the two numbers, propagating NaNs.
    ///
    /// This returns NaN when *either* argument is NaN, as opposed to
    /// [`f64::max`] which only returns NaN when *both* arguments are NaN.
```

(We don't mention unstable APIs in stable method documentation, but once this stabilizes then `f64::max` would get a similar note pointing here.)

Reply:
Done.